.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: "An issue or feature request related to the GUI"
+    url: https://github.com/bitcoin-core/gui/issues/new
```

Comment:
url should be like it was previously (`https://github.com/bitcoin-core/gui/issues/`), or include the `/new/choose` suffix to allow choosing a template